src/lightning/pytorch/strategies/strategy.py

```diff
@@ -152,6 +152,7 @@ def setup_precision_plugin(self) -> None:
         model, optimizers, lr_scheduler_configs = self.precision_plugin.connect(
             self.model, self.optimizers, self.lr_scheduler_configs
         )
+        model = self.precision_plugin.convert_module(model)
```

Comment:
What about doing it in connect? This would resolve https://github.com/Lightning-AI/lightning/issues/17655

Reply:
Yes that works too!